compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/util/Windows.desktop.kt

```diff
@@ -151,17 +162,3 @@ private val iconSize = Size(32f, 32f)
 internal fun Window.setIcon(painter: Painter?) {
     setIconImage(painter?.toAwtImage(density, layoutDirection, iconSize))
 }
-
-internal fun Window.makeDisplayable() {
-    val oldPreferredSize = preferredSize
-    val oldLocation = location
-    preferredSize = size
-    try {
```

Comment:
Why it was in `try` block? What possible exceptions it can throw? And why current calls are not wrapped anymore?

Reply:
Initially I used this rule here:

if you change a proprerty which you don't own, you should restore a previous value, and be cautious, because there can be an exception.

But if `pack` don't throw an exception, we can remove `try`

Reply:
Also, in the new code we don't restore the value, we just override `preferredSize` in the resize method. It is not needed there, as that method owns the size.